.travis.yml

```diff
@@ -100,6 +99,7 @@ matrix:
         - os: linux
           env: PYTHON_VERSION=3.5 NUMPY_VERSION=1.10
                INSTALL_CMD='python setup.py build_ext --inplace'
+               PIP_DEPENDENCIES='pytest-astropy'
```

Comment:
Is it needed ? It should be installed with astropy (`INSTALL_CMD`) ?

Reply:
It's needed because the install command won't install the test dependencies and since we aren't using the test command we can't tell the test runner to add packages from .eggs to the path.

Reply:
"install command won't install the test dependencies": oh, makes sense ! (sorry, it's late ;)).